packages/gestalt/src/Link.js

```diff
@@ -142,6 +151,11 @@ const LinkPropTypes = {
   >),
 };
 
-Link.propTypes = LinkPropTypes;
+function LinkWithRef(props, ref) {
+  return <Link {...props} forwardedRef={ref} />;
+}
+LinkWithRef.displayName = 'Link';
 
-export default Link;
+export default (React.forwardRef<Props, HTMLAnchorElement>(
```

Comment:
The component returned from `React.forwardRef` would no longer be named `Link`. I think we might want to do something like this:
```
const LinkWithForwardRef: React$AbstractComponent<Props, HTMLAnchorElement>) = React.forwardRef<Props, HTMLAnchorElement>(LinkWithRef);
LinkWithForwardRef.displayName = 'Link';
export default LinkWithForwardRef;
```